src/js/components/CosmosErrorMessage.js

```diff
@@ -71,7 +113,10 @@ CosmosErrorMessage.propTypes = {
   className: React.PropTypes.string,
   error: React.PropTypes.shape({
     message: React.PropTypes.string,
-    type: React.PropTypes.string
+    type: React.PropTypes.string,
+    data: React.PropTypes.shape({
+      errors: React.PropTypes.array
```

Comment:
@wavesoft note that the contents of the `data` field are dependent on the value of the `type` field in the error response. For `MarathonBadResponse`, this is [correct](https://github.com/dcos/cosmos/blob/525c58c94e3f5f61240418a104cdbb8f0a1d96c4/cosmos-server/src/main/scala/com/mesosphere/cosmos/CosmosError.scala#L65), but other error types have different fields. For example, `UnsupportedContentType` will have [`supported` and `actual` fields](https://github.com/dcos/cosmos/blob/525c58c94e3f5f61240418a104cdbb8f0a1d96c4/cosmos-server/src/main/scala/com/mesosphere/cosmos/CosmosError.scala#L83-L86) in its `data` object.